docs/100_producten/100_borgstelling/borgstelling-beeindigen.feature

```diff
@@ -1,14 +1,27 @@
 # language: nl
 Functionaliteit: Borgstelling beëindigen
 
-  Achtergrond: 
-    Gegeven het kenmerk "093e08db-6791-454b-a172-068099514907"
+  Achtergrond:
+    Gegeven een kredietbank
 
-  Scenario: Borgstelling beëindigen
+  Scenario: Borgstelling beëindigen van afgegeven aanvraag
```

Comment:
```suggestion
  Scenario: Borgstelling beëindigen
```
Ik twijfel of het volgende scenario een scenario is. Wanneer dat geen scenario ia dan kan deze aanvulling weg.